app/src/main.rs

```diff
@@ -56,9 +56,9 @@ extern "C" fn sample_main() {
         let settings_strings = &[["Blind signing", "Enable blind signing"]];
 
         loop {
-            let event = if tx_reviewer.nbgl_reviewer.display_settings {
+            let event = if tx_reviewer.inner.display_settings {
                 nbgl_display::<Ins>(&mut comm, settings_strings, 0)
-            } else if !tx_reviewer.nbgl_reviewer.review_started {
+            } else if !tx_reviewer.inner.review_started {
```

Comment:
If possible, let's not use `tx_reviewer.inner` outside the reviewer crate, and make `inner` private.

We could introduce two inline functions for `display_settings` and `review_started`